docs/examples/ex30.py

```diff
@@ -111,11 +111,9 @@ def dilatation(pressure: np.ndarray) -> np.ndarray:
 
 velocity = flow(pressure)
 
-basis['psi'] = InteriorBasis(mesh, ElementQuad2())
+basis['psi'] = InteriorBasis(mesh, ElementQuad2(), intorder=3)
 psi = np.zeros(A.shape[0])
-vorticity = asm(rot, basis['psi'],
-                w=[basis['psi'].interpolate(velocity[i::2])
-                   for i in range(2)])
+vorticity = asm(rot, basis['psi'], w=basis['u'].interpolate(velocity))
```

Comment:
Why isn't this also required in ex18?

https://github.com/kinnala/scikit-fem/blob/6e31b46e1d11d12d93798405d38c3764a18a8f72/docs/examples/ex18.py#L83-L85

which still seems to work.